Readme.md

```diff
@@ -113,6 +114,78 @@ $ make test
   - [Julian Gruber](https://github.com/juliangruber)
   - [Yiyu He](https://github.com/dead-horse)
 
+
+## Backers
+
+Support us with a monthly donation and help us continue our activities. [[Become a backer](https://opencollective.com/koajs#backer)]
```

Comment:
Then maybe we should remove the first backer icon/link? It just seems a bit pushy.